### PR TITLE
reverts to linting on all files & no docs linting

### PR DESCRIPTION
**Pull request by dpaiton**

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -36,18 +36,18 @@ jobs:
       - name: Run black
         run: python -m black --check --diff .
 
-      - name: Get changed Python files
-        id: changed_files
+      - name: Get all Python files
+        id: list_files
         run: |
-          echo "files=$(git diff --name-only HEAD^ HEAD | grep '.py$' | xargs)" >> $GITHUB_OUTPUT
+          echo "::set-output name=files::$(git ls-files '*.py' '*.pyi')"
```
Comment (wakamex) [inline .github/workflows/lint.yml:42]: set-output is deprecated in favor of `echo "{name}={value}" >> $GITHUB_OUTPUT` https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Comment (dpaiton) [reply .github/workflows/lint.yml:42]: hmm ok I tried that in the first commit and it failed. I'll try again.

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -47,7 +47,7 @@ jobs:
           token: ${{ secrets.GH_GITHUB_COM_TOKEN || github.token}}
       - run: |
           python -m pip install --upgrade pip
-          python -m pip install .[with-dependencies,postgres,ape]
+          python -m pip install .[with-dependencies,postgres]
```
Comment (wakamex) [inline .github/workflows/test.yml:50]: we no longer have a postgres dependency section. should we just add it to `with-dependencies`?

File: .github/workflows/lint.yml (hunk 1 above)
Comment (dpaiton) [reply .github/workflows/lint.yml]: yeah it is failing now:
```
Run echo "files=$(git ls-files '*.py' '*.pyi')" >> $GITHUB_OUTPUT
Error: Unable to process file command 'output' successfully.
Error: Invalid format 'elfpy/__init__.py'
```

Comment (dpaiton) [reply .github/workflows/lint.yml]: yeah it's a whole thing :( 
https://stackoverflow.com/questions/74137120/how-to-fix-or-avoid-error-unable-to-process-file-command-output-successfully

Comment (dpaiton) [reply .github/workflows/lint.yml]: xargs worked

**Review by sentilesdal — APPROVED** [review]

**Pull request merged**